backend/graminsta/post/models.py

```diff
@@ -12,9 +13,12 @@ class Post(models.Model):
     publisher = models.ForeignKey(auth_models.User, on_delete=models.CASCADE,
                                   related_name="published_posts")
     description = models.TextField()
-    img = models.ImageField(upload_to='img')
+    location = models.TextField()
```

Comment:
I think maybe you can use two `double` fields for lng, lat instead?

Reply:
Changed to lat and lng